package/kernel/linux/modules/netsupport.mk

```diff
@@ -1376,6 +1376,7 @@ $(eval $(call KernelPackage,mpls))
 define KernelPackage/9pnet
   SUBMENU:=$(NETWORK_SUPPORT_MENU)
   TITLE:=Plan 9 Resource Sharing Support (9P2000)
+  DEPENDS:=+!LINUX_6_6:kmod-fs-netfs
```

Comment:
Please add upstream reference patch link or build error log for **all** missing dependency commits. It's not hard to track them by querying the corresponding path git history.

Reply:
OK